Tests.NHibernate.Spatial.MsSql2008/packages.config

```diff
@@ -1,10 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="GeoAPI" version="1.7.4" targetFramework="net40-client" />
-  <package id="Iesi.Collections" version="4.0.1.4000" targetFramework="net40-Client" />
-  <package id="log4net" version="2.0.7" targetFramework="net40-client" />
-  <package id="NetTopologySuite" version="1.14" targetFramework="net40-client" />
-  <package id="NetTopologySuite.IO" version="1.14.0.1" targetFramework="net40-client" />
-  <package id="NHibernate" version="4.1.1.4000" targetFramework="net40-client" allowedVersions="[4,5)" />
-  <package id="NUnit" version="2.6.4" targetFramework="net40-Client" />
+  <package id="Antlr3.Runtime" version="3.5.1" targetFramework="net461" />
+  <package id="GeoAPI" version="1.7.4" targetFramework="net461" />
+  <package id="Iesi.Collections" version="4.0.2" targetFramework="net461" />
+  <package id="log4net" version="2.0.8" targetFramework="net461" />

```

Comment:
For tests projects, I have also upgraded log4net by the way (from 2.0.7), being quite a minor change.
But I have not touched to NUnit, leaving it at its 2.x version.

Reply:
👍